embc-app/ViewModels/Registration.cs

```diff
@@ -1,21 +1,21 @@
-using AutoMapper;
+using AutoMapper;
 using Gov.Jag.Embc.Public.Utils;
 using Newtonsoft.Json;
-using System;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using static Gov.Jag.Embc.Public.Models.Db.Enumerations;
 
-namespace Gov.Jag.Embc.Public.ViewModels
-{
+namespace Gov.Jag.Embc.Public.ViewModels
+{
     public class RegistrationMappingProfile : Profile
     {
         public RegistrationMappingProfile()
         {
             CreateMap<Registration, Models.Db.EvacueeRegistration>()
                 .ForMember(d => d.EssFileNumber, opts => opts.MapFrom(s => s.EssFileNumber))
```

Comment:
Can be deleted

Reply:
Line 17?

Reply:
Yes, notice that AutoMapper will map both by convention as the name is the same. Line 18 I'm not sure, you can test with and without and see if it mapped by convention or not